README.adoc

```diff
@@ -0,0 +1,61 @@
+== Spring-Boot Camel QuickStart
```

Comment:
Karaf not Spring Boot :slightly_smiling_face:

Reply:
ups, I"ll fix it (copy paste)